x-pack/plugins/observability/public/pages/rule_details/index.tsx

```diff
@@ -39,25 +36,23 @@ import {
   RuleEventLogListProps,
 } from '@kbn/triggers-actions-ui-plugin/public';
 // TODO: use a Delete modal from triggersActionUI when it's sharable
-import { ALERTS_FEATURE_ID } from '@kbn/alerting-plugin/common';
-
+import { ALERTS_FEATURE_ID, RuleExecutionStatusErrorReasons } from '@kbn/alerting-plugin/common';
 import { AlertConsumers } from '@kbn/rule-data-utils';
 import { DeleteModalConfirmation } from '../rules/components/delete_modal_confirmation';
 import { CenterJustifiedSpinner } from '../rules/components/center_justified_spinner';
-import { OBSERVABILITY_SOLUTIONS } from '../rules/config';
+import { getHealthColor, OBSERVABILITY_SOLUTIONS } from '../rules/config';
```

Comment:
@fkanout Just FYI once my [Rules PR](https://github.com/elastic/kibana/pull/134173) is merged, you will notice that I moved `getHealthColor` from rules folder to rule_details folder https://github.com/elastic/kibana/pull/134173/files#diff-f162f9f5c3f157fced238bfb00cca2ed679ddbc28db1449352803b3374c677dfR11, since Rules page doesn't need this anymore (I get everything for free from shareable rules table)

Reply:
done